From d2e57789574bc0e6ccec55c49cad267e28b95e58 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Wed, 10 Apr 2024 16:46:17 +0200
Subject: [PATCH 0724/1695] dts: stm32: default disable TAMP node in stm32mp2x
 SoC device tree files

Default disable TAMP node and activate it at board level.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 603a106709da389fe1221e3ad21ebbdd857f4948 dts: stm32: default disable TAMP node in stm32mp2x SoC device tree files]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: I8133f7c0d9e8518992c84095c9662ff3db9a74cb
---
 core/arch/arm/dts/stm32mp211.dtsi | 1 +
 core/arch/arm/dts/stm32mp231.dtsi | 1 +
 core/arch/arm/dts/stm32mp251.dtsi | 1 +
 3 files changed, 3 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp211.dtsi b/core/arch/arm/dts/stm32mp211.dtsi
index 529c4cdb9..a6e02e5c3 100644
--- a/core/arch/arm/dts/stm32mp211.dtsi
+++ b/core/arch/arm/dts/stm32mp211.dtsi
@@ -670,6 +670,7 @@
 			#size-cells = <1>;
 			ranges;
 			st,backup-zones = <24 24 24 24 12 12 8>;
+			status = "disabled";
 
 			nvram: nvram@46010100 {
 				compatible = "st,stm32mp25-tamp-nvram";
diff --git a/core/arch/arm/dts/stm32mp231.dtsi b/core/arch/arm/dts/stm32mp231.dtsi
index 45b265c06..19e511fe9 100644
--- a/core/arch/arm/dts/stm32mp231.dtsi
+++ b/core/arch/arm/dts/stm32mp231.dtsi
@@ -969,6 +969,7 @@
 			#address-cells = <1>;
 			#size-cells = <1>;
 			ranges;
+			status = "disabled";
 
 			nvram: nvram@46010100 {
 				compatible = "st,stm32mp25-tamp-nvram";
diff --git a/core/arch/arm/dts/stm32mp251.dtsi b/core/arch/arm/dts/stm32mp251.dtsi
index ad3be3f55..74818333f 100644
--- a/core/arch/arm/dts/stm32mp251.dtsi
+++ b/core/arch/arm/dts/stm32mp251.dtsi
@@ -1024,6 +1024,7 @@
 			#address-cells = <1>;
 			#size-cells = <1>;
 			ranges;
+			status = "disabled";
 
 			nvram: nvram@46010100 {
 				compatible = "st,stm32mp25-tamp-nvram";
-- 
2.47.2

